lms/static/sass/course/_info.scss

```diff
@@ -69,6 +69,19 @@ div.info-wrapper {
       margin-bottom: lh();
       padding-left: 0;
 
+      .show-older-updates {
+        -webkit-border-radius: 5px;
+        -moz-border-radius: 5px;
+        border-radius: 5px;
```

Comment:
`border-radius` should be fine, according to http://caniuse.com/#search=border-radius. No need for the prefixed versions.